content/en/style-guide/_index.md

```diff
@@ -176,6 +178,91 @@ More about that in the [How To Contribute](/contribute/) doc.
 Before getting started, please read some published Glossary terms 
 to get a feeling for the level of detail and difficulty and when examples are appropriate.
 
+
+
+## Sign Language Video Style Guide {#sign-language-style-guide}
+
+
+### If you are contributing a sign language video, please follow these guidelines to ensure our videos are clear, informative, and accessible to a wide audience (Including hearing community members).
```

Comment:
I don't think this part needs to be in bold.

Reply:
made the change. wyt?